liquibase-core/src/main/java/liquibase/datatype/core/BooleanType.java

```diff
@@ -19,11 +20,18 @@ public class BooleanType extends LiquibaseDataType {
     @Override
     public DatabaseDataType toDatabaseDataType(Database database) {
         String originalDefinition = StringUtil.trimToEmpty(getRawDefinition());
-        if ((database instanceof Firebird3Database)) {
+        if ((database instanceof FirebirdDatabase)) {
+            try {
```

Comment:
As a non-developer, I find the use of a try/catch instead of if/else interesting. If anyone has time, can you please explain the advantages of this approach? I'm always happy to learn something new. :)

Reply:
@nvoxland Do we want to log the exception??

Reply:
I added a log of the exception